blog/posts/announcing-melange-5.md

```diff
@@ -0,0 +1,285 @@
+
+```js
+[].push(3);
+```
+
+Besides being more versatile, having an explicit marker with `@mel.this` is
+also more visually intuitive: when scanning Melange code containing `external`
+bindings, it becomes easier to spot which is the "this" argument. This feature
+is 100% backwards compatible with `@mel.send`: in the absence of `@mel.this`,
+the instance argument defaults to the first one declared in the signature, as
+previously supported.
+
+## Additional quality of life improvements
+
+### OCaml 5.3 Compatibility / Stdlib Upgrade
+
+Since [Melange's
+inception](https://anmonteiro.com/2021/03/on-ocaml-and-the-js-platform/), one
+of its goals has been to keep it up to date with the latest OCaml releases.
+[This
+release](https://github.com/melange-re/melange/releases/download/5.0.1-53/melange-5.0.1-53.tbz)
+brings Melange up to speed with OCaml 5.3, including upgrades to the `Stdlib`
+library as well. We're also releasing Melange 5 for OCaml
+[4.14](https://github.com/melange-re/melange/releases/download/5.0.1-414/melange-5.0.1-414.tbz),
+[5.1](https://github.com/melange-re/melange/releases/download/5.0.1-51/melange-5.0.1-51.tbz)
+and
+[5.2](https://github.com/melange-re/melange/releases/download/5.0.1-52/melange-5.0.1-52.tbz).
+
+### Melange runtime NPM packages
+
+Starting from this release, we're shipping NPM packages with the precompiled
+Melange runtime. This feature, requested by a few users in
+[melange#620](https://github.com/melange-re/melange/issues/620) allows to use
+Melange without compiling its own runtime and stdlib (essentially, in
+combination with `(emit_stdlib false)` in `(melange.emit ..)`).
+
+This can be useful in monorepos that compile multiple Melange applications but,
+perhaps most importantly, it enables Melange libraries and packages to be
+published without the weight of the full runtime / stdlib.
+
+### Better editor support for Melange `external`s
+
+Melange bindings to JavaScript, specified through `external` declarations, used
+to propagate internal information in the [native
+payload](https://ocaml.org/manual/5.3/intfc.html#external-declaration). In
+practice, hovering over one of these in your editor could end up looking a bit
+weird:
+
+![](https://private-user-images.githubusercontent.com/661909/386930735-af1c5c90-2ba7-4813-b4d8-80a4f9a74aae.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEzOTE4MjIsIm5iZiI6MTc0MTM5MTUyMiwicGF0aCI6Ii82NjE5MDkvMzg2OTMwNzM1LWFmMWM1YzkwLTJiYTctNDgxMy1iNGQ4LTgwYTRmOWE3NGFhZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMzA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDMwN1QyMzUyMDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xYTVmNDA3Nzk4MjhlNGEzNjE0MmVkMGVlOTM3NDgzMzM4MzUwYWVlNjJjNmMxMmY1NjU2Mjg4MjM2NGYyYTI0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.T6wIwTPOmHyWpcCBcJoon1d42nQjMGzqch7P2BbRI3Q)
```

Comment:
I am not able to see these images in the rendered version. I think we'll have to create some `blog/posts/images` folder and use relative links for this to work?